src/Lasso.jl

```diff
@@ -190,26 +193,35 @@ function Base.show(io::IO, path::LassoPath)
     Base.showarray(io, [path.λ path.pct_dev ncoefs]; header=false)
 end
 
-StatsBase.coef(path::LassoPath) = path.coefs
-
 ## MODEL CONSTRUCTION
 
 # Controls early stopping criteria with automatic λ
 const MIN_DEV_FRAC_DIFF = 1e-5
 const MAX_DEV_FRAC = 0.999
 
 # Compute automatic λ values based on X'y and λminratio
-function computeλ(Xy, λminratio, α, nλ)
+function computeλ(Xy, λminratio, α, nλ, ω::@compat(Union{Vector,Void}))
     λmax = abs(Xy[1])
-    for i = 1:length(Xy)
+    if ω != nothing
```

Comment:
It should be a little more efficient if this is written as `!isa(ω, Void)` here and below.